src/ExperimentalCycles.mo

```diff
@@ -1,51 +1,152 @@
-/// Managing cycles
+/// Managing cycles within actors on the Internet Computer (IC).
 ///
-/// Usage of the Internet Computer is measured, and paid for, in _cycles_.
+/// The usage of the Internet Computer is measured, and paid for, in _cycles_.
 /// This library provides imperative operations for observing cycles, transferring cycles and
```

Comment:
```suggestion
/// This library provides imperative operations for observing cycles, transferring cycles, and
```

Oxford comma? https://external-preview.redd.it/GQtcAhbyy90XcRpBCc7co6j13457lA1jlfaZG7R0C3g.jpg?auto=webp&s=c453a24dec2337fbed8e714c7f7ba1e3d0f28a09

Reply:
Indeed, thanks for the suggestion.